### PR TITLE
Logs converter

### PR DESCRIPTION
**Pull request by PNieck**
## Description

PR contains script to convert logs form binary forma to `csv` and the other way around.

## Motivation and Context

Easier logs manimulation

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [x] Tested by hand on: (list targets here). `host-generic-pc`

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing linter checks and tests passed.
- [x] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [x] I will merge this PR by myself when appropriate.

### CODE REVIEW

**Comment by PNieck** [comment]
Codespell is not recognizing `ned` keyword.

File: scripts/logs/common/formats/binary/bin_parser.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            north=self.__parse_field(file, field),
+            east=self.__parse_field(file, field),
+            down=self.__parse_field(file, field)
+        )
+
+    def __parse_vector3(self, file: BufferedReader, field: FieldSpecifier) -> utils.Vector3:
+        return utils.Vector3(
+            x=self.__parse_field(file, field),
+            y=self.__parse_field(file, field),
+            z=self.__parse_field(file, field)
```
Comment (niewim19) [inline scripts/logs/common/formats/binary/bin_parser.py:104]: nitpic: no spaces around `=`

File: scripts/logs/common/formats/binary/bin_export.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+from common.formats.binary.fields import FIELDS
+
+
+class BinaryLogExporter(LogsVisitor):
+    def __init__(
+            self,
+            byte_order: Literal['little', 'big'] = 'little'
+    ) -> None:
+        self.byte_order = byte_order
+        self.fields_specifiers = FIELDS
```
Comment (niewim19) [inline scripts/logs/common/formats/binary/bin_export.py:13]: The same idea as in `bin_parser.py` about direct `FIELDS.ger("acceleration")` usage

**Review by niewim19 — CHANGES_REQUESTED** [review]

File: scripts/logs/common/formats/binary/bin_parser.py (hunk 1 above)
Comment (PNieck) [reply scripts/logs/common/formats/binary/bin_parser.py:104]: Yes, but according to our Python codding conventions we don't use spaces around `=`, when it comes to function arguments.

Comment (niewim19) [reply scripts/logs/common/formats/binary/bin_parser.py]: Yeah, missed that. Remark is invalid :)

**Review by niewim19 — APPROVED** [review]
Very nicely written, good job!

**Pull request merged**